FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -47,39 +47,67 @@
 
 /**
  Similar to our much-loved XCTAssert() macros. Use this to perform your test. No need to write an explanation, though.
- @param view The view to snapshot
+ @param view The view to snapshot.
  @param identifier An optional identifier, used if there are multiple snapshot tests in a given -test method.
- @param suffixes An NSOrderedSet of strings for the different suffixes
- @param tolerance The percentage of pixels that can differ and still count as an 'identical' view
+ @param suffixes An NSOrderedSet of strings for the different suffixes.
+ @param tolerance The percentage of pixels that can differ and still count as an 'identical' view.
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__, tolerance__) \
     FBSnapshotVerifyViewOrLayerWithOptions(View, view__, identifier__, suffixes__, tolerance__)
 
+/**
+ Similar to our much-loved XCTAssert() macros. Use this to perform your test. No need to write an explanation, though.
+ @param view The view to snapshot.
+ @param identifier An optional identifier, used if there are multiple snapshot tests in a given -test method.
+ @param suffixes An NSOrderedSet of strings for the different suffixes.
+ @param pixelTolerance The percentage a pixel's R,G,B and A components can differ and still be considered 'identical'.
```

Comment:
I think it would be clearer if it said "The percentage a given pixel's..." in the headerdoc places that this is needed

Reply:
Done 👍